api-service/controller/alerts_api.go

```diff
@@ -83,18 +83,18 @@ func (ctrl *APIController) SearchAlerts(w http.ResponseWriter, r *http.Request)
 	}
 
 	//get the data
-	hosts, err := ctrl.Service.SearchAlerts(mode, search, sortBy, sortDesc, pageNumber, pageSize, severity, status, from, to)
+	response, err := ctrl.Service.SearchAlerts(mode, search, sortBy, sortDesc, pageNumber, pageSize, severity, status, from, to)
 	if err != nil {
 		utils.WriteAndLogError(ctrl.Log, w, http.StatusInternalServerError, err)
 		return
 	}
 
+	//Write the data
 	if pageNumber == -1 || pageSize == -1 {
-		//Write the data
-		utils.WriteJSONResponse(w, http.StatusOK, hosts)
+		utils.WriteJSONResponse(w, http.StatusOK, response)
 	} else {
-		//Write the data
-		utils.WriteJSONResponse(w, http.StatusOK, hosts[0])
+		alerts := response[0]
+		utils.WriteJSONResponse(w, http.StatusOK, alerts)
```

Comment:
Why don't write `utils.WriteJSONResponse(w, http.StatusOK, alerts)` instead?

Reply:
I think you mean `utils.WriteJSONResponse(w, http.StatusOK, response[0])`.
The goal of this name change is documentation, in fact it is not clear at all that `response[0]` contains alerts.
I think this renaming clarify the goal of that `if` and what's happening, at zero performance costs.

Reply:
Ok